samples/dnn/audio_visual_speech_recognition.py

```diff
@@ -0,0 +1,239 @@
+from collections import deque
+import cv2
+import numpy as np
+import argparse
+
+'''
+AVSpeechRecognition
+
+    How to obtain the model required for this sample:
+    Option 1: Download the model from https://drive.google.com/file/d/1xuwk5ZQagKFoTXev27zvlSg8dAmSJoo7/view?usp=sharing
+    Option 2: Convert the model using pretrained torch model and base repo.
+        Use the colab notebook here: https://colab.research.google.com/drive/1awBCZ5O6uAT32cHvufNWad5m6q26TuqQ?usp=sharing
+'''
+
+class AVSpeechRecognition:
+    '''
+    Audio Video Speech Recognition based on AVHubert (arXiv:2201.02184 [eess.AS])
+    '''
+    def __init__(self, source, type='camera', detector_path='face_detection_yunet_2022mar.onnx', model_path = 'AVSpeechRecog.onnx', margin=20, video_width=640, video_height=480, score_threshold=0.9, nms_threshold=0.3, top_k=5000, show_video=False):
```

Comment:
It's enough to specify default detector_path, model_path, show_video, etc. in argparser only